### PR TITLE
Add CloudAMQP

### LINKED ISSUE #6712: Add CloudAMQP
### Site name

CloudAMQP

### Site URL

https://www.cloudamqp.com/

### Documentation URL

https://www.cloudamqp.com/blog/two-factor-authentication.html

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] FIDO2/U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 sites on [Similarweb](https://www.similarweb.com/) or has a substantial social media following.
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by person1234-ux**

### CODE REVIEW

**Comment by Carlgo11** [comment]
@person1234-ux if you want to link an issue with a PR you can write any of the following messages in the PR description:
```
Closes #6712
```
```
Resolves #6712
```
```
Fixes #6712
```
Those are keywords letting GitHub know that the PR and issue are related.

Also, as all other PRs that add a site use the title format `Add <site name>` you might want to consider using that for future PRs. It makes going through multiple PRs easier for us :wink:

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I ran the logo through [TinyPNG](https://tinypng.com) and reduced the image quite a lot. I also suggest you rename it to `cloudamqp.com.png` to follow the naming scheme.

File: entries/c/cloudamqp.com.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "CloudAMQP": {
+    "domain": "cloudamqp.com",
+    "img": "cloudamqp-logo.png",
+    "tfa": [
+      "totp",
+      "sms"
+      
+    ],
+    "documentation": "https://www.cloudamqp.com/blog/two-factor-authentication.html",
+    "notes": "Sms is required as a backup method",
```
Comment (Carlgo11) [inline entries/c/cloudamqp.com.json]: As SMS is an abbreviation for Short Messaging Service, all letters should be capitalized.
```suggestion
    "notes": "SMS is required as a backup method",
```

File: entries/c/cloudamqp.com.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "CloudAMQP": {
+    "domain": "cloudamqp.com",
+    "img": "cloudamqp-logo.png",
+    "tfa": [
+      "totp",
+      "sms"
+
```
Comment (Carlgo11) [inline entries/c/cloudamqp.com.json]:
```suggestion
```

File: entries/c/cloudamqp.com.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "img": "cloudamqp-logo.png",
+    "tfa": [
+      "totp",
+      "sms"
+      
+    ],
+    "documentation": "https://www.cloudamqp.com/blog/two-factor-authentication.html",
+    "notes": "Sms is required as a backup method",
+    "keywords": [
+      "cloud",
```
Comment (Carlgo11) [inline entries/c/cloudamqp.com.json]: From what I understand RabbitMQ is a message-broker and not cloud computing so the cloud category should be removed.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/c/cloudamqp.com.json (hunk 3 above)
Comment (person1234-ux) [reply entries/c/cloudamqp.com.json]: ok

**Review by person1234-ux — COMMENTED** [review]
done

**Comment by phallobst** [comment]
Here is an SVG version of the rabbit logo:
cloudamqp com

**Comment by person1234-ux** [comment]
note to self update to use svg instead of png

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**